Utilities/DataLoading/EdFi.LoadTools/ApiClient/OdsRestClient.cs

```diff
@@ -39,7 +39,12 @@ public OdsRestClient(IApiConfiguration configuration, OAuthTokenHandler tokenHan
             _tokenHandler = tokenHandler;
             _schemaNames = schemaNames;
 
-            _httpClient = new HttpClient
+            HttpClientHandler handler = new HttpClientHandler
+            {
+                ServerCertificateCustomValidationCallback = (sender, cert, chain, sslPolicyErrors) => true
```

Comment:
Same question here, this is a security risk.

Reply:
ODS-4720